### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by pmaria**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
   "exports": "./build/src/main.js",
   "version": "0.0.0-semantically-released",
   "description": "A pipeline for the NDE Dataset Knowledge Graph",
+  "engines": {
+    "node": ">=16"
```
Comment (ddeboer) [inline package.json:8]: 20 is fine here, which is LTS now:

```suggestion
    "node": ">=20"
```

File: test/graphdb.test.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  let client: GraphDBClient;
+
+  beforeAll(async () => {
+    startedGraphDb = await graphDb.start();
+
+    const formData = new FormData();
+    formData.append(
+      'config',
+      fs.createReadStream(
+        path.join(__dirname, './assets/graphdb-repo-config.ttl')
```
Comment (ddeboer) [inline test/graphdb.test.ts:35]: Should this be checked in?

Comment (pmaria) [reply test/graphdb.test.ts:35]: yep, that explains the not working test :)

**Comment by ddeboer** [comment]
@pmaria Is this now ready for review or still a draft?

**Comment by pmaria** [comment]
@ddeboer ready. Did you manage with the uriSpace test?

**Comment by ddeboer** [comment]
> @ddeboer ready. Did you manage with the uriSpace test?

Sorry for the wait. As it turns out, the implementation worked fine (really?! 😆). There are two analyzers at play:

1. a plain SPARQL analyser that returns object URI prefixes
2. which is then wrapped in the uriSpace analyzer proper:

https://github.com/netwerk-digitaal-erfgoed/dataset-knowledge-graph/blob/c7225ad76f480c2cad4ce2abe0b947933484cac8/src/main.ts#L49-L51

This second analyzer compares the URI prefixes against the known list of prefixes from the Network of Terms catalog. The uriSpace test was testing 2, but assuming the implementation of 1. I changed the test fixture to refer to a terminology source from that catalog (AAT) and the relevant assertions.

**Pull request merged**